00-prerequisites.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: First Steps in LHCb
+subtitle: Pre-workshop Checklist
+minutes: 10
+---
+> ## Learning Objectives {.objectives}
+>
+> * Check you are ready for the workshop
+
+Please read and try the following steps **before** arriving. For
+some of the steps the solution is easy, but requires waiting for
+a day or so. So please try them before arriving and try to fix
+what ever does not work.
+
+This will be an interactive workshop, so you will need to bring
+a computer. There will be no machines for you to use in the room.
+
+If this is the first time you are bringing your laptop to CERN, you
+will have to [register it](https://network.cern.ch) before it can
+access the internet.
+
+Please bring an ethernet/network cable (even if you have wifi) and
+your power supply.
+
+> In the following we assume you use Mac OS or Linux, unfortunately
+> we do not know how to do these things on Windows. If you do know,
+> please contact us (lhcb-starterkit@cern.ch) so we can update these instructions.
+
+Try the following steps with the computer you will use at the workshop:
+
+1. In your browser, on the computer you are planning to bring to the
```

Comment:
In that case, I think it should be said in every point... Either you say it at the top, and it covers all points, or you say it in every point.